gravitee-apim-console-webui/src/management/api/analytics/pathMappings/api-path-mappings.model.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Api } from '../../../../entities/api';
+
+export interface PathMappingDS {
+  path: string;
+}
+
+export interface ApiPathMappingsEditDialogData {
```

Comment:
Juste a note : Personally i add this kind of type with then dialog component to have it just next to the  `@Inject(MAT_DIALOG_DATA) dialogData ...` i found it more easier to maintain